### PR TITLE
Create Blog Post “using-markdown-and-github-for-blogging-part-1”

### PR DESCRIPTION
**Pull request by hongsups**
Automatically generated by Netlify CMS

### CODE REVIEW

File: content/blog/posts/using-markdown-and-github-for-blogging-part-1.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+### From pull request to merge
+
+Once the usual back-and-forth review process begins, reviewers make comments and requests for changes, and authors either accept them or make rebuttal. This process will be documented through comments and every piece of communication is tracked. Once everyone is satisfied, we make a collective decision to merge the pull request and it becomes a part of the master branch. Once this is done, we have several options. We can use several existing services to publish our repository directly as a website, or do something more sophisticated.
+
+## Coming soon
+
+In Part 2, our volunteer software engineer, [Nick Holden](https://github.com/nholden), will explain how we decided to use our existing content management system (CMS) to host the blog and publish posts. This approach provides consistency in development, and convenience in publication.
```
Comment (nholden) [inline content/blog/posts/using-markdown-and-github-for-blogging-part-1.md:72]: Breaking this into two separate blog posts makes a lot of sense to me, and I'm happy to write about the topic you describe here! I'm not sure it's necessary to preview it here though. What do you think about leaving this bit off? We can always add a link to part 2 here once it's published.

**Review by nholden — APPROVED** [review]
Love it! Great work. 👏👏👏

File: content/blog/posts/using-markdown-and-github-for-blogging-part-1.md (hunk 1 above)
Comment (hongsups) [reply content/blog/posts/using-markdown-and-github-for-blogging-part-1.md:72]: Sure that works! I will leave this part off. Thanks for the feedback, Nick :)

**Comment by hongsups** [comment]
Just wanted to confirm that using CMS to modify the existing manuscript just updates the same PR. :)

**Pull request merged**